app/forms/two_factor_options_form.rb

```diff
@@ -32,12 +32,17 @@ def extra_analytics_attributes
     }
   end
 
+  def selection_is_voice_or_sms
+    %w[voice sms] & selection
+  end
```

Comment:
is this method supposed to return a boolean  or an array? based on the name, I'd expect a boolean, but `&` on two arrays returns another array

I feel like I'd expect something more like:
```suggestion
  def selection_is_voice_or_sms?
    (%w[voice sms] & selection).present?
  end
```